### DISCUSSION TITLE
Custom menu sidebar to include icon

### DISCUSSION THREAD

**Discussion by sophielamb** [thegetty/quire#679] [Get Help] [answered]
When we used Quire V0, Materia took inspiration from the Nelson-Atkins Museum of Art to mark essays in the side menu with an icon to denote those which were peer reviewed. We'd be very keen to replicate this in Quire V1, but I have very little JavaScript under my belt to make this happen.

In V0, we added a yaml key-value for peer review, either true or false. Greg then kindly helped us to add in code to say 'if peer review is true, then the icon is displayed by the title'.

How would this work in V1? Would the process be similar?

![materia2titlepage](https://user-images.githubusercontent.com/69150936/222547002-5dbdf43f-d939-418a-8c97-e49c852088fd.png)

![materia2partials-menu-header-crop](https://user-images.githubusercontent.com/69150936/222547236-c2d96e2b-90cc-48a8-8917-86b1fec1869e.png)

![materia2-contents-list-crop](https://user-images.githubusercontent.com/69150936/222547321-37e2e63d-5d6e-4f3b-ad7c-fd808714c354.png)

![materia2campmd-crop](https://user-images.githubusercontent.com/69150936/222547472-0e23d4cd-b81b-47ff-9c4b-cde3eaf2ceb2.png)

**Comment by geealbers** [comment, marked as answer]
@sophielamb sorry for the delay, but I was finally able to carve some time out to tackle this. It also inspired us to start a new section of the documentation, called [_Recipes_](https://quire.getty.edu/resources/recipes/), where we'll share step-by-step lessons on how to make interesting and useful customizaitions to Quire projects.

And the answer to your question is our first _Recipe_: https://quire.getty.edu/resources/recipes-menu-icon/. If you end up using it, please let us know how it goes, if it works for you, or if there are ways the directions could be more clear. I'll be interested to hear your feedback.

And as an aside, I've struggled myself getting up to speed with heavy use Quire makes of JavaScript now. And while there's still a lot I don't understand, I've found that it's starting to make more sense and that much of the structure of the logic is feeling familiar, it's primarily the syntax/language that's different. I've also appreciated that finding the answers to JavaScript questions online is _much_ easier than for Go templates, and that the JavaScript experience I'm learning here is more broadly applicable to the world outside Quire. Still some of it is complete gibberish to be sure. :)

**Reply by Erin-Cecele** [reply]
Tagging @materiajournal to take over for @sophielamb
